components/script/timers.rs

```diff
@@ -136,9 +186,66 @@ impl OneshotTimers {
             suspended_since: Cell::new(None),
             suspension_offset: Cell::new(Duration::ZERO),
             expected_event_id: Cell::new(TimerEventId(0)),
+            runsteps_active: DomRefCell::new(FxHashMap::default()),
```

Comment:
Nit: `Default::default()` for the whole expression (same on next line)

Reply:
Good point.